Package.swift

```diff
@@ -5,29 +5,26 @@ import PackageDescription
 
 let package = Package(
     name: "MoEngage-iOS-SDK",
-    products: [
-        // Products define the executables and libraries a package produces, and make them visible to other packages.
-        .library(
-            name: "MoEngage-iOS-SDK",
-            targets: ["MoEngageCore","MoEngageSDK","MoEngageAnalytics","MoEngageMessaging","MoEngageObjCUtils", "MoEngageSecurity"])
-    ],
-    dependencies: [
-        // Dependencies declare other packages that this package depends on.
-        // .package(url: /* package url */, from: "1.0.0"),
-    ],
-    targets: [
-        .binaryTarget(name: "MoEngageSecurity", path: "./Frameworks/MoEngageSecurity.xcframework"),
-        
-        .binaryTarget(name: "MoEngageCore", path: "./Frameworks/MoEngageCore.xcframework"),
-        
-        .binaryTarget(name: "MoEngageAnalytics", path: "./Frameworks/MoEngageAnalytics.xcframework"),
-        
-        .binaryTarget(name: "MoEngageMessaging", path: "./Frameworks/MoEngageMessaging.xcframework"),
-        
-        .binaryTarget(name: "MoEngageObjCUtils", path: "./Frameworks/MoEngageObjCUtils.xcframework"),
+    platforms: [.iOS(.v11), .tvOS(.v11)],
+    products: [], dependencies: [], targets: [],
+    swiftLanguageVersions: [.v5]
+)
+
+let version = "9.16.2"
+let targetMap = [
```

Comment:
With this change, is there any improvement in the integration time ?

Reply:
This will improve package fetch time as all the versions of xcframeorks won't be downloaded, instead only xcframeowrks associated with particular version will be downloaded. However the benifit will only be applicable to newer versions released after this change not the older versions.